src/views/ApartmentApp/components/StudentApplication/components/BottomBar/index.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { Backdrop, Button, Card, CardContent, Grid, Typography } from '@material-ui/core/';
+import GordonLoader from '../../../../../../components/Loader';
+import GordonDialogBox from '../../../../../../components/GordonDialogBox';
+import SaveButton from './components/SaveButton';
+
+const submitAlertText = (
+  <span>
+    This feature is not yet implemented.
```

Comment:
As discussed with @EjPlatzer , I will be moving the `ApartmentHeader` stuff into this component, for various logistical and stylistic reasons. I guess I will do that right now, because the "Delete" option will likely need to go in here as well

Reply:
Also the key detail is that in the picture, that was not the bottom bar, that was the ApartmentHeader object but pinned at the bottom. A merged version will be pushed in about 15 minutes or less